### PR TITLE
fixed pending publication issue noted in #4261

### PR DESCRIPTION
**Pull request by hyi**
### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]

### CODE REVIEW

**Comment by hydrocheck** [comment]
| METRIC | VALUE |
|:-----:|:-----:|
| NEW SECURITY HOTSPOTS | 0 |
| NEW CODE SMELLS | 0 |
| NEW VULNERABILITIES | 0 |
| NEW BUGS | 0 |
| NEW DUPLICATED LINES DENSITY | 0.0 |

[https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4262](https://sonarqube.cuahsi-workstation.com:9000/dashboard?id=hydroshare-4262)

File: django_irods/views.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def download(request, path, use_async=True, use_reverse_proxy=True,
     6. a previously zipped file that was zipped asynchronously.
 
     """
-    if __debug__:
```
Comment (sblack-usu) [inline django_irods/views.py:56]: There are tons of other places we are using `__debug__`.  If we remove the if statement, we can update the logging settings to write the debug logs for the package as needed.  We aren't writing DEBUG logs by default in production anyways.

File: hs_core/tasks.py [hunk 2]
```diff
@@ -90,7 +89,7 @@ def manage_task_nightly():
             if response.status_code == status.HTTP_200_OK:
                 # retry of metadata deposition succeeds, change resource flag from failure
                 # to pending
-                res.doi = get_resource_doi(act_doi, 'pending')
+                res.doi = act_doi
```
Comment (sblack-usu) [inline hs_core/tasks.py:92]: Why was this change made?  It seems unrelated to the __debug__ changes.

**Comment by hyi** [comment]
Closing this PR I created against develop branch since it was covered by the hotfix PR here: https://github.com/hydroshare/hydroshare/pull/4278.
